community/urls.py

```diff
@@ -18,7 +19,7 @@ def get_index():
 
 urlpatterns = [
     distill_url(
-        r'^$', TemplateView.as_view(template_name='index.html'),
+        r'^$', overview_index,
```

Comment:
with this change, the file `templates/index.html` becomes unused.  But we need to be using more raw html & templates, and less code generated HTML (ugly).

Instead of replacing `index.html` I think it would be better for the twitter feed to be on `/twitter/` , in a new app in `/twitter` directory (like activity)

Then the `/twitter` link can be added to  `templates/index.html` .